### PR TITLE
Update to Bootstrap Icons 11.1.1 and upgrade to Typescript 5

### PR DESCRIPTION
**Pull request by tobiasquadflieg**

### CODE REVIEW

File: rollup.config.js [hunk 1]
```diff
@@ -1,44 +1,44 @@
-import typescript from 'rollup-plugin-typescript2';
-import { terser } from 'rollup-plugin-terser';
+import terser from "@rollup/plugin-terser";
```
Comment (tommyip) [inline rollup.config.js:1]: Can we not change `'` to `"`

File: package.json [hunk 2]
```diff
@@ -26,21 +27,23 @@
     "url": "https://github.com/tommyip/bootstrap-icons-vue/issues"
   },
   "scripts": {
-    "build": "node ./src/codegen.js && rollup -c",
+    "build": "node ./src/codegen.cjs && rollup -c",
```
Comment (tommyip) [inline package.json:30]: Is there a need to add `"type": "module",` and `cjs`?

**Review by tommyip — COMMENTED** [review]
Thanks for the work, just some minor comments.

File: rollup.config.js (hunk 1 above)
Comment (tobiasquadflieg) [reply rollup.config.js:1]: Sure, we can change it, if u like. I just used my Prettier in VSCode to do some code styling. Maybe we should create a prettier conf to keep the styling streamlined.

File: package.json (hunk 2 above)
Comment (tobiasquadflieg) [reply package.json:30]: Not sure, i had some issues with the migration to typescript 5 and rollup, could be a change that is not needed.

**Comment by tommyip** [comment]
Released to npm.

**Pull request merged**